freeipa_community_portal/config.py

```diff
@@ -27,7 +27,7 @@
 class Config(object):
     development_config = ('freeipa_community_portal/conf/'
                           'freeipa_community_portal_dev.ini')
-    deployment_config = '/etc/freeipa_community_portal.ini'
+    deployment_config = '/etc/ipa/extensions/community-portal/portal.ini'
```

Comment:
We can get rid of freeipa-portal-install as soon as we have RPM packages.

For create-portal-user I'm planning to use freeipa_community_portal.config in the script.